packages/sanity/src/_internal/cli/commands/app/deployCommand.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  type CliCommandArguments,
+  type CliCommandContext,
+  type CliCommandDefinition,
+} from '@sanity/cli'
+
+import {type DeployStudioActionFlags} from '../../actions/deploy/deployAction'
+
+const helpText = `
+Options
+  --source-maps Enable source maps for built bundles (increases size of bundle)
+  --auto-updates / --no-auto-updates Enable/disable auto updates of studio versions
+  --no-minify Skip minifying built JavaScript (speeds up build, increases size of bundle)
+  --no-build Don't build the application prior to deploy, instead deploying the version currently in \`dist/\`
+  -y, --yes Unattended mode, answers "yes" to any "yes/no" prompt and otherwise uses defaults
+
+Examples
+  sanity deploy
+  sanity deploy --no-minify --source-maps
+`
+
+const appDeployCommand: CliCommandDefinition = {
+  name: 'deploy',
+  group: 'app',
+  signature: '[SOURCE_DIR] [--no-build]  [--source-maps] [--no-minify]',
```

Comment:
Is the extra whitespace expected? 
```suggestion
  signature: '[SOURCE_DIR] [--no-build] [--source-maps] [--no-minify]',
```

Reply:
also fixed it in the file I copy/pasted from 😅